components/Card/Card.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import styled from "styled-components";
+
+interface Props extends React.Attributes {
+  className?: string;
+  style?: React.CSSProperties;
+  children?: React.ReactNode;
+  onClick?: () => void;
+  hoverable?: boolean;
+}
+
+export default function Card({
+  children,
+  onClick,
+  hoverable,
+  ...props
+}: Props) {
+  const CardDiv = styled.div`
```

Comment:
`Root` is preferred, probably?

```suggestion
  const Root = styled.div`
```